### DISCUSSION TITLE
Is there any way to run a script from pip

### DISCUSSION THREAD

**Discussion by Swey9** [bazel-contrib/rules_python#1701] [Q&A]
I want to run Cheetah3 using bazel, however for linux users, Cheetah3 defines script instead of an entry point. I've tried using `py_console_script_binary` which didn't work since it doesn't have any entry points. Is there any way to do that for scripts?

**Comment by Swey9** [comment]
Managed to do that by supplying `py_console_script_binary` my own "entry_points.txt", in which I created entry points for Cheetah3 based on its scripts.
Maybe it would be nice if we could use the scripts directly from the whl's data, and maybe it should be better documented

**Reply by aignas** [reply]
Right now we only support console_script standard as documented in the rules.

Could you describe what cheetah3 does? Does it just include the script as whl data? What's the complete setup here?
